routers/install/install.go

```diff
@@ -390,7 +390,7 @@ func SubmitInstall(ctx *context.Context) {
 	cfg.Section("database").Key("LOG_SQL").SetValue("false") // LOG_SQL is rarely helpful
 
 	cfg.Section("repository").Key("ROOT").SetValue(form.RepoRootPath)
-	cfg.Section("server").Key("SSH_DOMAIN").SetValue(form.Domain)
+	cfg.Section("server").Key("SSH_DOMAIN").MustString(form.Domain)
```

Comment:
Are we really going to convert each `SetValue` to `MustString` in a separate PR now?

Reply:
Do you have a better idea for this PR?

Reply:
That wasn't meant as disapproval, I've simply noticed a tendency lately to replace `SetValue` in this method with `MustString`.
I'm wondering how many other settings we will convert as well in subsequent PRs.